src/components/tile-select/tile-select.pw.tsx

```diff
@@ -0,0 +1,610 @@
+      mount,
+      page,
+    }) => {
+      await mount(<AccordionTileSelectComponent accordionExpanded={false} />);
+
+      const getElementInput = getElement(
+        page,
+        "tile-select-accordion-content"
+      ).locator("..");
+      await expect(getElementInput).not.toBeVisible();
+    });
+
+    test(`should check when accordionExpanded set as true`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<AccordionTileSelectComponent accordionExpanded={true} />);
+
+      const getElementInput = getElement(
+        page,
+        "tile-select-accordion-content"
+      ).locator("..");
+      await expect(getElementInput).toBeVisible();
+    });
+
+    testData.forEach((accordionContent) => {
+      test(`should check when accordionContent set as ${accordionContent}`, async ({
+        mount,
+        page,
+      }) => {
+        await mount(
+          <TileSelectComponent accordionContent={accordionContent} />
+        );
+
+        const getElementInput = getElement(
+          page,
+          "tile-select-accordion-content"
+        );
+        await expect(getElementInput).toHaveText(accordionContent);
+      });
+    });
+  });
+
+  test.describe("check props for MultiTileSelect component", () => {
+    test(`should check when type prop is set as checkbox`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<TileSelectComponent multiSelect type="checkbox" />);
+
+      await expect(inputElement(page)).toHaveAttribute("type", "checkbox");
+    });
+
+    test(`should check when type prop is set as radio`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<TileSelectComponent multiSelect type="radio" />);
+
+      await expect(inputElement(page)).toHaveAttribute("type", "radio");
+    });
+
+    test(`should check multiSelect prop`, async ({ mount, page }) => {
+      await mount(<MultiTileSelectGroupComponent />);
+
+      const multiSelectInputElement0 = inputElement(page).nth(0);
+      await multiSelectInputElement0.click();
+      await expect(multiSelectInputElement0).toHaveAttribute(
+        "aria-checked",
+        "true"
+      );
+      const multiSelectInputElement1 = inputElement(page).nth(1);
+      await multiSelectInputElement1.click();
+      await expect(multiSelectInputElement1).toHaveAttribute(
+        "aria-checked",
+        "true"
+      );
+      const multiSelectInputElement2 = inputElement(page).nth(2);
+      await expect(multiSelectInputElement2).toHaveAttribute(
+        "aria-checked",
+        "false"
+      );
+      const multiSelectInputElement3 = inputElement(page).nth(3);
+      await expect(multiSelectInputElement3).toHaveAttribute(
+        "aria-checked",
+        "false"
+      );
+    });
+
+    test(`should render with the expected border radius styling`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<TileSelectComponent />);
+
+      const ele = inputElement(page).locator("..");
```

Comment:
**suggestion(non-blocking):** this could be given a better variable name. Something like `inputElementChildren` is a bit more descriptive.